autosklearn/automl.py

```diff
@@ -917,6 +936,168 @@ def refit(self, X, y):
+            elif task in REGRESSION_TASKS:
+                include['regressor'] = self._include_estimators
+            else:
+                raise ValueError(task)
+        elif self._exclude_estimators is not None:
+            if task in CLASSIFICATION_TASKS:
+                exclude['classifier'] = self._exclude_estimators
+            elif task in REGRESSION_TASKS:
+                exclude['regressor'] = self._exclude_estimators
+            else:
+                raise ValueError(task)
+
+        # Prepare missing components to the TAE function call
+        if 'include' not in kwargs:
+            kwargs['include'] = include
+        if 'exclude' not in kwargs:
+            kwargs['exclude'] = exclude
+        if 'memory_limit' not in kwargs:
+            kwargs['memory_limit'] = self._memory_limit
+        if 'resampling_strategy' not in kwargs:
+            kwargs['resampling_strategy'] = self._resampling_strategy
+        if 'metric' not in kwargs:
+            kwargs['metric'] = self._metric
+        if 'disable_file_output' not in kwargs:
+            kwargs['disable_file_output'] = self._disable_evaluator_output
+        if 'pynisher_context' not in kwargs:
+            kwargs['pynisher_context'] = self._multiprocessing_context
+        if 'stats' not in kwargs:
+            scenario_mock = unittest.mock.Mock()
+            scenario_mock.wallclock_limit = self._time_for_task
+            kwargs['stats'] = Stats(scenario_mock)
+        kwargs['stats'].start_timing()
+
+        # Allow to pass the cutoff also as an argument
+        cutoff = kwargs.pop('cutoff', self._per_run_time_limit)
+
+        # Fit a pipeline, which will be stored on disk
+        # which we can later load via the backend
+        ta = ExecuteTaFuncWithQueue(
+            backend=self._backend,
+            autosklearn_seed=self._seed,
+            abort_on_first_run_crash=False,
+            cost_for_crash=get_cost_of_crash(kwargs['metric']),
+            port=self._logger_port,
+            **kwargs,
+            **self._resampling_strategy_arguments
+        )
+
+        run_info, run_value = ta.run_wrapper(
+            RunInfo(
+                config=config,
+                instance=None,
+                instance_specific=None,
+                seed=self._seed,
+                cutoff=cutoff,
+                capped=False,
+            )
+        )
+
+        pipeline = None
+        if run_value.status == StatusType.SUCCESS:
+            if kwargs['resampling_strategy'] in ('cv', 'cv-iterative-fit'):
+                load_function = self._backend.load_cv_model_by_seed_and_id_and_budget
+            else:
+                load_function = self._backend.load_model_by_seed_and_id_and_budget
+            try:
+                pipeline = load_function(
+                    seed=self._seed,
+                    idx=run_info.config.config_id + 1,
+                    budget=run_info.budget,
+                )
+            except Exception as e:
```

Comment:
What kind of exceptions are you thinking can happen here? From looking at the code we should not catch them but rather let them go to the user because right now the user would not be able to know which exception happens in a programmatic fashion.